### PR TITLE
explicitly time unit tests and print to logs

### PR DESCRIPTION
**Pull request by mikedanese**
To make collecting stats eaiser.

ref https://github.com/kubernetes/kubernetes/issues/25940

### CODE REVIEW

**Comment by MHBauer** [comment]
My only suggestion would be to provide a formatting flag to the `date` command so we know for sure what format the date is in.

File: hack/test-go.sh [hunk 1]
```diff
@@ -324,7 +324,12 @@ for (( i=0, j=0; ; )); do
   etcdPrefix=${etcdPrefixes[j]}
   echo "Running tests for APIVersion: $apiVersion with etcdPrefix: $etcdPrefix"
   # KUBE_TEST_API sets the version of each group to be tested.
-  KUBE_TEST_API="${apiVersion}" ETCD_PREFIX=${etcdPrefix} runTests "$@"
+  time {
+   kube::log::status "Starting tests at '$(date)'"
```
Comment (ixdy) [inline hack/test-go.sh]: `kube::log::status` includes a timestamp, so $(date) is a little redundant, though not that big a deal.

File: hack/test-go.sh [hunk 2]
```diff
@@ -324,7 +324,12 @@ for (( i=0, j=0; ; )); do
   etcdPrefix=${etcdPrefixes[j]}
   echo "Running tests for APIVersion: $apiVersion with etcdPrefix: $etcdPrefix"
```
Comment (ixdy) [inline hack/test-go.sh]: maybe just make this a `kube::log::status` instead of echo?

**Comment by ixdy** [comment]
what's the status of this?

**Pull request closed without merging**